noah/src/main/scala/edu/uci/ics/cloudberry/noah/news/NewsCollection.java

```diff
@@ -0,0 +1,174 @@
+package edu.uci.ics.cloudberry.noah.news;
+
+/**
+ * Collecting news articles from webhose.io and storing in JSON file.
+ *
+ * Created by Kaiyi Ma on 7/12/16.
+ */
+
+import com.buzzilla.webhose.client.WebhoseClient.WebhoseUrl;
+import com.buzzilla.webhose.client.WebhoseQuery;
+import com.buzzilla.webhose.client.WebhoseResponse;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.ObjectWriter;
+import com.google.api.client.http.*;
+import com.google.api.client.http.javanet.NetHttpTransport;
+import com.google.api.client.json.JsonFactory;
+import com.google.api.client.json.JsonObjectParser;
+import com.google.api.client.json.jackson2.JacksonFactory;
+import org.apache.commons.io.FileUtils;
+import org.apache.http.NameValuePair;
+import org.apache.http.client.utils.URLEncodedUtils;
+
+import java.io.*;
+import java.nio.charset.StandardCharsets;
+import java.util.List;
+import java.util.Properties;
+
+
+public class NewsCollection {
+
+    private static String apiKey = "";
+    private static String filePath = "";
+    private static String ts = null;
+    private static String propFileName = System.getProperty("user.dir") + "/noah/src/main/scala/edu/uci/ics/cloudberry/noah/news/news.properties";
```

Comment:
you should access it by the resource path other than the system path. @aishwaryakapse  knows how to access it.